### DISCUSSION TITLE
Why did version 1.6.17 delete the spawn point for transporting guests?

### DISCUSSION THREAD

**Discussion by KougamiYuriko** [FakeFishGames/Barotrauma#14900] [Bug reports]
Every time I restart the game, the NPC will randomly appear at a spawn point, such as standing with the captain. Sometimes it will appear in a room without oxygen and die from lack of oxygen in a few minutes.

**Comment by Ek-Videogames** [comment]
VIP & Prisoner spawn points were removed as well in unstable.

**Comment by Regalis11** [comment]
This seems like an oversight: we didn't intend to remove these. We'll look into it, sorry for the inconvenience!

**Comment by NilanthAnimosus** [comment]
Closing as fixed and merged. Should be in a future hotfix
